frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -70,6 +89,95 @@ const CampOverviewPage = (): JSX.Element => {
 
   const openManageSessionsModal = () => onOpen();
```

Comment:
ik not part of this PR but can't we rename the original onOpen function to be used as openManageSessionsModal? 

see like 62, instead of `const { isOpen, onOpen, onClose } = useDisclosure();` we can do 
```
const { isOpen: manageSessionsIsOpen, onOpen: manageSessionsOnOpen, onClose: : manageSessionsOnClose } = useDisclosure();
```